### PR TITLE
Network May release

### PR DESCRIPTION
**Pull request by anton-evseev**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by number213** [comment]
@MyronFanQiu could you please review & merge? This is a monthly release for Network

**Comment by MyronFanQiu** [comment]
@number213 Please fix LintDiff CI. Thanks.

**Review by mmyyrroonn — APPROVED** [review]

**Comment by number213** [comment]
@MyronFanQiu done in https://github.com/Azure/azure-rest-api-specs/pull/9787, please approve and merge

**Comment by MyronFanQiu** [comment]
Merged this PR since all changes have been reviewed by ARM team.

File: specification/network/resource-manager/readme.python.md [hunk 1]
```diff
@@ -59,6 +60,7 @@ batch:
 ```yaml $(multiapi) && $(track2)
 clear-output-folder: true
 batch:
+  - tag: package-2020-04
```
Comment (MyronFanQiu) [inline specification/network/resource-manager/readme.python.md:63]: @number213 We don't add package-2020-05 into this one

**Pull request merged**